addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -141,7 +141,7 @@ export class HtmlField extends Component {
                         // Ensure all external links are opened in a new tab.
                         retargetLinks(this.readonlyElementRef.el);
 
-                        const hasReadonlyModifiers = Boolean(this.props.record.activeFields[this.props.fieldName].modifiers.readonly);
+                        const hasReadonlyModifiers = Boolean(this.props.record.isReadonly(this.props.fieldName));
```

Comment:
cc @julien-banken I think I remember you showing me a change that you planned to make in your branch that was related to this.

Reply:
When loading a record with the `Record` component of  `@web/views/record`, the `modifiers` property was `undefined` and an error was raised because `undefined` does not have any `readonly` property. To solve that issue, I planned to do a small safe-check like this:

```js
const field = this.props.record.activeFields[this.props.fieldName];
const hasReadonlyModifiers = field.modifiers && Boolean(field.modifiers.readonly)
if (!hasReadonlyModifiers) {
    // ...
}
```

If we have a clean and safe `isReadonly` method on the record, that should be better than doing what I did :slightly_smiling_face:

Branch: https://github.com/odoo-dev/odoo/tree/master-knowledge-template-gallery-jbn